### ISSUE TITLE
SNOW_n doesn't work if phases are not labelled 1,2,3 etc

### ISSUE THREAD

**Issue by TomTranter** [PMEAL/porespy#168]
In the test changing self.im3d+1 to +2 fails the test as there is a bug in partitioning
```python
    def test_snow_n(self):
        net = ps.networks.snow_n(self.im3d+2, voxel_size=1,
                                 boundary_faces=['left', 'right', 'front',
                                                 'back', 'top', 'bottom'],
                                 marching_cubes_area=True)
        found_nans = False
        for key in net.keys():
            if np.any(np.isnan(net[key])):
                found_nans = True
        assert found_nans is False
```

**Comment by TomTranter** [comment]
Either it should accept any set of numbers or there should be a re-labeling step or the docs should be updated to explain this behavior and an error message should be thrown. I would prefer accepting any set of numbers as then grey-scale values can be used directly

**Comment by Zohaib-Atiq** [comment]
This will require modifying five different function in snow_n. The code will also look ugly after this modification. Also, setting up alias's will create confusion. I guess using make contiguous function before inputting your image is a good idea. This can be done inside the snow_n code as well. I will also change the doc string for more convenience. If all agree.

**Comment by jgostick** [comment]
After today's discussion, it would be preferable if the function could accept any numbers into the alias argument like {0: solid, 25: void, 127: binder}.  However, if this ends up being complicated, I think it's reasonable to expect users to preprocess their image to make the labels contiguous.  

This is also relating to #246

**Comment by Zohaib-Atiq** [comment]
@jgostick this has been resolved in porespy-private PR #13

**Comment by jgostick** [comment]
Fixed in #265
